apps/graphql/src/apps/itinerary/types/outputs/Itinerary.js

```diff
@@ -37,6 +38,15 @@ export default new GraphQLObjectType({
     startTime: { type: GraphQLDateType },
     type: { type: GraphQLString },
     bookingToken: { type: GraphQLString },
+    checked: {
```

Comment:
Could you name it `isChecked` and `isValid` because it's a boolean prop?

Reply:
Sure, remaned.